### ISSUE TITLE
missing information for screen reader users

### ISSUE THREAD

**Issue by rilygreat** [HabitRPG/habitrpg-ios#1299]
> The following lines help us find and squash the Bug you encountered. Please do not delete/change them.
> iOS Version: 16.5
> Device: iPhone15,3
> App Version: 3.7.2 (869)
> User ID: 3eaa4d67-f70c-438c-bdf8-2a2ac926c00c
> Level: 322
> Class: warrior
> Is in Inn: false
> Uses Costume: true
> Custom Day Start: 2
> 
> Please describe the bug you encountered:
> I would also like to report that there are several bits of missing information in the app for screen reader users. The number of days a daily has been checked off is the biggest one (this also applies to the habits tab, making habits somewhat useless for screen reader users). In the Warrior class, when using the action that allows you to maintain your streak color as blue, the tasks are not given their values in the menu, requiring the user to manually check in Dailies and To-Dos, remember what task is at what color value, go into the skills menu, then find the task,  having to repeat this set of extra steps until the task value is where they want it. By contrast, a sighted user need only tap the skill and then tap the task to watch it change color.

**Comment by AriPearce** [comment]
Hi! I would like to work on this issue (in particular the day count for habits/dailies). My Habitica user is @aripepear.

Thanks!

**Comment by saraolson** [comment]
@AriPearce you're welcome to take this on if you're still interested!

**Comment by AriPearce** [comment]
On it @saraolson!
